docs/samples/calling/index.html

```diff
@@ -37,6 +37,8 @@ <h2 class="collapsible">Authentication</h2>
                     require token with spark:calls_write scope.</p>
                   <button id="enableProduction" type="button" class="btn-code" onclick=changeEnv()>In Production</button><span class="text-color"> ( Click to change the environment )</span>
                   <input id="access-token" name="accessToken" placeholder="Access Token" value="" type="text" style="margin: 1rem 0 0.5rem 0;" required>
+
+                  <input id="jwt-token-for-dest" class="hidden" name="jwtToken" placeholder="JWT token for destination" value="" type="text" style="margin: 0.5rem 0 0.5rem 0;" required>
```

Comment:
Just saying JWT  or JWE for placeholder and id should be enough. Also, we should update the access token placeholder for the guest to say Guest Token

Reply:
1. Updated the placeholder accordingly
  - Guest: Guest Token
  - Normal: Access Token
2. Just saying JWT/JWE wouldn't be little confusing? JWT tokens are used as access token as well. Let me know if this change still needs to be made.

Reply:
Placeholder didn't update for me, please check once